IdentityCore/src/broker_operation/request/browser_native_message_request/MSIDBrowserNativeMessageGetCookiesRequest.m

```diff
@@ -58,4 +65,15 @@ - (NSDictionary *)jsonDictionary
     @throw MSIDException(MSIDGenericException, @"Not implemented.", nil);
 }
 
+#pragma mark - Private
+
+- (NSString *)uriHost
+{
+    if (!self.uri) return nil;
+    
+    NSURL *url = [[NSURL alloc] initWithString:self.uri];
```

Comment:
how about empty

Reply:
it is possible to create url with empty string, url.host will be nil in such case, so we are safe here.